messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/AbstractMessagingDatabusConsumer.java

```diff
@@ -299,6 +295,15 @@ protected String getReaderNameSuffix() {
 
   protected abstract void createPartitionReaders() throws IOException;
 
+  public synchronized Long getPendingData() throws Exception {
```

Comment:
I can't see an external API exposed in the top-level interface MessageConsumer? Note that almost all client applications call MessageConsumerFactory.create(...) which returns MessageConsumer instance.